.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+labels: modelserver
```

Comment:
Maybe we should consider adding the label 'enhancement' here?

Reply:
Good idea, thanks!